src/app.vue

```diff
@@ -23,10 +23,8 @@ onMounted(async () => {
     await account.setActiveProducts()
     if (accountErrors.value?.length > 0) { return }
   }
-  const identifier = useRoute().params.identifier as string
-  await useBcrosBusiness().loadBusiness(identifier)
```

Comment:
moved into the dashboard onbeforemount (will have already triggered inside the layout anyways though)